### ISSUE TITLE
Flatpak version doesn't start on Ubuntu 21.04 (Xorg)

### ISSUE THREAD

**Issue by asamofal** [flameshot-org/flameshot#1762]
Hello!

**Flameshot version**
Flatpak 0.10.0

**Describe the bug**
Can't start the tool.

**To Reproduce**
`flatpak run org.flameshot.Flameshot gui`

```
Jul 26 14:22:27 laptop systemd[2796]: Started Application launched by gsd-media-keys.
Jul 26 14:22:27 laptop systemd[2796]: Started app-flatpak-org.flameshot.Flameshot-41951.scope.
Jul 26 14:22:27 laptop systemd[2796]: app-gnome-flatpak-41951.scope: Succeeded.
Jul 26 14:22:27 laptop dbus-daemon[2812]: [session uid=1000 pid=2812] Activating service name='org.flameshot.Flameshot' requested by ':1.380' (uid=1000 pid=41959 comm="/usr/libexec/flatpak-dbus-proxy --args=42 ")
Jul 26 14:22:27 laptop systemd[2796]: Started app-flatpak-org.flameshot.Flameshot-41966.scope.
Jul 26 14:22:27 laptop flameshot[41974]: Failed to load module "canberra-gtk-module"
Jul 26 14:22:27 laptop flameshot[41974]: Failed to load module "canberra-gtk-module"
Jul 26 14:22:27 laptop org.flameshot.Flameshot[41974]: Qt: Session management error: None of the authentication protocols specified are supported
Jul 26 14:22:27 laptop org.flameshot.Flameshot[41974]: QSettings::value: Empty key passed
Jul 26 14:22:27 laptop org.flameshot.Flameshot[41974]: QSettings::value: Empty key passed
Jul 26 14:22:27 laptop org.flameshot.Flameshot[41974]: QSettings::setValue: Empty key passed
Jul 26 14:22:27 laptop org.flameshot.Flameshot[41974]: QSettings::value: Empty key passed
Jul 26 14:22:27 laptop org.flameshot.Flameshot[41974]: QSettings::setValue: Empty key passed
Jul 26 14:22:27 laptop dbus-daemon[2812]: [session uid=1000 pid=2812] Successfully activated service 'org.flameshot.Flameshot'
Jul 26 14:22:27 laptop systemd[2796]: app-flatpak-org.flameshot.Flameshot-41951.scope: Succeeded.
Jul 26 14:22:27 laptop gnome-shell[3056]: Window manager warning: Invalid WM_TRANSIENT_FOR window 0x480000a specified for 0x4800008.
Jul 26 14:22:27 laptop systemd[1]: systemd-localed.service: Succeeded.
Jul 26 14:22:28 laptop kernel: [14212.737078] flameshot[41974]: segfault at 8 ip 00007f9812206025 sp 00007ffec8365dc0 error 4 in libQt5Widgets.so.5.15.3[7f98121f5000+3a7000]
Jul 26 14:22:28 laptop kernel: [14212.737091] Code: 83 e0 bf c1 e6 06 09 c6 40 88 b7 f0 00 00 00 e9 81 f4 ff ff 90 c3 90 66 66 2e 0f 1f 84 00 00 00 00 00 0f 1f 00 41 55 41 54 55 <4c> 8b 67 08 48 89 f5 4d 8d 6c 24 68 4c 89 ef e8 a7 79 ff ff 84 c0
Jul 26 14:22:28 laptop systemd[2796]: app-flatpak-org.flameshot.Flameshot-41966.scope: Succeeded.
```

**System Information**

Ubunt 21.04
GNOME Shell 3.38.4
Flatpak 1.11.2
Linux 5.13.5-051305-generic
xserver-xorg-core 2:1.20.11-1ubuntu1.1

**Comment by ibeex** [comment]
related to https://github.com/flathub/org.flameshot.Flameshot/issues/6

**Comment by mmahmoudian** [comment]
@asamofal the fix (v0.10.1) is on the way. I will close this. Feel free to open this if the issue persisted in the new version
